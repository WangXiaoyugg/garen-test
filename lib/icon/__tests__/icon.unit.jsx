import * as renderer from "react-test-renderer"
import React from 'react'
import Icon from '../icon'
// import {mount} from 

describe("test icon component", () => {
    it("render successfully", () => {
        const json = renderer.create(<Icon name="qq"/>).toJSON()
        expect(json).toMatchSnapshot()
    })
})