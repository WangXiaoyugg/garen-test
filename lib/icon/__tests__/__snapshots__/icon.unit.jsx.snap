// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test icon component render successfully 1`] = `
<svg
  className="gui-icon"
>
  <use
    xlinkHref="#qq"
  />
</svg>
`;
